spec/webdrivers/chrome/driver_remote_version_finder_spec.cr

```diff
@@ -7,7 +7,8 @@ module Webdrivers::Chrome
       File.delete(finder.cache_path) if File.exists?(finder.cache_path)
     end
 
-    it "will limit return version to matching major version if version" do
+    # TODO: Figure out if this is necessary
+    pending "will limit return version to matching major version if version" do
```

Comment:
This is actually failing, but I have no idea what it's really needed for, or the best way to go about fixing it.

Reply:
Looks like it was connected the `latest_release` method you removed and yea, it's necessary. We have to find the latest chromedriver version that matches the version of the user's browser. Also, this test was named poorly. It should be "will limit return version to matching major version if version provided" or something like that.

Reply:
Oh, I see. Yeah, when I ran it locally, it said expected version 71 but got version 118. I guess I can bump that version in the spec to a later one, but it feels like that will eventually be out of date again. [This endpoint](https://googlechromelabs.github.io/chrome-for-testing/known-good-versions.json) only goes down to 113. I'd imagine that it will drop off at some point, so if we said "expect v113" and it goes away, the spec will start failing. 

None of these have a "latest release", but the one I'm using is "Stable" which is the latest Stable release version. They got rid of https://googlechromelabs.github.io/chrome-for-testing/LATEST_RELEASE so you have to specify which version of the latest release you want like https://googlechromelabs.github.io/chrome-for-testing/LATEST_RELEASE_118 (which downloads a text file)

I'm not sure the best route to take here. Any suggestions?

Reply:
Ahh, the ruby gem does a condition based on which version it's looking for https://github.com/titusfortner/webdrivers/blob/main/lib/webdrivers/chromedriver.rb#L65

Personally, I'd like to go that route as well since docker containers are likely to not update their chrome even if they update this shard. We'd just have to add specs to test both paths.  I would have used a web mocking shard if there was a good one available, but I'm not sure how to handle this new way that drops old versions off the page in a spec that we want to pass in a year from now.